yarn.lock

```diff
@@ -1570,7 +1571,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"bignumber.js@npm:^9.0.1":
+"bignumber.js@npm:^9.0.0, bignumber.js@npm:^9.0.1":
```

Comment:
This is because some of our deps also depend on bignum, right?

Reply:
I believe so.